### ISSUE TITLE
Adaptive aspect ratio

### ISSUE THREAD

**Issue by sashabeep** [metafizzy/flickity#904]
**Test case:** https://codepen.io/anon/pen/wNEmWp#anon-login

Why, whyyyyyyyyyyyyyy is always 480px height????????? Why are every time i should use media queries to set size of gallery????? How to do it auto? It's very basic feature. All the images are same size and aspect!

![2019-02-14 14 57 08](https://user-images.githubusercontent.com/5689796/52785367-d9c7e900-3068-11e9-9084-df6ba5ef07d3.png)

**Comment by desandro** [comment]
Add [imagesLoaded](https://flickity.metafizzy.co/options.html#imagesloaded) and :

``` css
.carousel-cell img {
  display: block;
  max-width: 100%;
}
```

See demo https://codepen.io/desandro/pen/d2fff607ae8b4b68178bc62e9ab52a61

**Comment by godot882** [comment]
@sashabeep You will have to change a few things to make this work. I've created a fork of your pen with the modifications:

https://codepen.io/godot/pen/WPmqvV

Note the `imagesLoaded` and `setGallerySize` properties, and the css applied to the `.outer` class. Also note that I've inserted your whole html inside another wrapper, this is for the "aspect-ratio css" to work.

If you have any questions about the approach and why it behaves the way it does, feel free to ask.
